tools/bowtie2.wdl

```diff
@@ -0,0 +1,355 @@
+version 1.1
+
+import "../data_structures/read_group.wdl"
+
+# Bowtie2 accepts several function parameters.
+# The first term is a function type. Available function types are:
+# - C - constant
+# - L - linear
+# - S - square-root
+# - G - natural logarithm
+# The constant and coefficient types may be negative and/or floating point numbers.
+#
+# An example input JSON entry might look like:
+# ```
+# {
+#     "interval_seed_substrings": {
+#         "function_type": "S",
+#         "constant": 1,
+#         "coefficient": 0.50,
+#     },
+# }
+# ```
+# See the function documentation in bowtie2
+# (https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#setting-function-options)
```

Comment:
Can we upgrade this to preamble comments? (Looking forward to the v1.2 upgrade when we can make these a mutli-line string in the stuct `meta` section 🤞 )